docs/manual/job-options.md

```diff
@@ -589,6 +590,10 @@ remote values URL. The property reference is of the form
 like "http://server/options?option2=${option.option2.value}", then that option
 will depend on the value of the "option2" option.
 
+If the `“option2”` value have a unsafe character as `“My value”` the unsafe character 
+will be encoded to `"My%20value"` and the URL will be "http://server/options?My%20value",
+the po JSON must be stored as `“My%20value”` on the server side or the request will return a 404 error.
```

Comment:
What is `po`?

Reply:
Sorry, this is a misspelling it was intended to be pointed "the pointed JSON"